Tests/AutoTests/FunctionalTest/functional-ADLogin.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test, expect, chromium, Page, Browser, BrowserContext } from '@playwright/test';
+
+import * as app from "../../Configuration/appConfig.json";
+import loginPage from '../../pageObject/Login.page';
+
+test.describe("AD Authentication Sign in For The maritime-safety-information", () => {
+  let login: loginPage;
+  test.beforeEach(async ({ page }) => {
+    await page.goto(app.rnwAdminUrl);
+    login = new loginPage(page);
+  });
+
+  test('With the Valid details', async ({ page, context }) => {
```

Comment:
Can we please have a test to check what happens for invalid credentials, and also unauthorised users not in the correct role?

Reply:
resolved